### PR TITLE
 #167826408 Add filter by origin functionality

### PR DESCRIPTION
**Pull request by nziokaivy**
#### What does this PR do?
Adds filter by origin functionality.

#### Description of Task to be completed?
- Add filter trip dropdown with all, origin, destination
- Add styling to filter trip dropdown
- Remove unnecessary spaces
- Add blank line add the end of every file

#### How should this be manually tested?
Clone this repository and run `npm install && npm run test`

#### Any background context you want to provide?
Creating a filter by origin functionality enables the users to be able to view trips filtered by the origin of their choice.

#### What are the relevant pivotal tracker stories?
 #167826408

#### Screenshots (if appropriate)
Screenshot from 2019-08-12 16-33-17

### CODE REVIEW

**Review by oesukam — CHANGES_REQUESTED** [review]
@nziokaivy Try to format your files to remove unnecessary space

**Comment by nziokaivy** [comment]
> @nziokaivy Try to format your files to remove unnecessary space

@oesukam Noted

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "no-restricted-syntax": ["error"],
+      "quotes": ["error", "single", { "allowTemplateLiterals": true }],
+      "implicit-arrow-linebreak": 0,
+      "consistent-return": 0,
+      "no-unused-vars": [
+      "error", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }
+      ],
+      "eol-last": ["error", "always"]
+  }
+}
```
Comment (oesukam) [inline .eslintrc.json:31]: @nziokaivy Blank line required at the end of the file

File: UI/html/admin.html [hunk 2]
```diff
@@ -143,53 +128,42 @@ <h1>Trips</h1>
...
-            </table>    
-        </section>    
-    </section>        
+            </table>
+        </section>
+    </section>
 </body>
-</html>
 
+</html>
```
Comment (oesukam) [inline UI/html/admin.html]: @nziokaivy Blank line required at the end of the file

File: UI/html/index.html [hunk 3]
```diff
@@ -123,26 +130,24 @@ <h1>Contact Us</h1>
...
-        <a href="https://google.com" class="google" target="_blank"><i class="fa fa-google-plus"></i></a> 
+        <a href="https://facebook.com" class="facebook" target="_blank"><i class="fa fa-facebook"></i></a>
+        <a href="https://twitter.com" class="twitter" target="_blank"><i class="fa fa-twitter"></i></a>
+        <a href="https://google.com" class="google" target="_blank"><i class="fa fa-google-plus"></i></a>
         <a href="https://linkedin.com" class="linkedin" target="_blank"><i class="fa fa-linkedin"></i></a>
-        <a href="https://youtube.com" class="youtube" target="_blank"><i class="fa fa-youtube"></i></a> 
+        <a href="https://youtube.com" class="youtube" target="_blank"><i class="fa fa-youtube"></i></a>
     </div>
-    
 </body>
```
Comment (oesukam) [inline UI/html/index.html]: @nziokaivy Blank line required at the end of the file

File: UI/html/newtrip.html [hunk 4]
```diff
@@ -32,21 +31,17 @@ <h2>Add New Trip</h2>
...
             <input type="text" placeholder="Price">
             <button class="btn">Add Trip</button>
-            </form>
+        </form>
     </div>
-        
-
 </body>
-</html>
+</html>
```
Comment (oesukam) [inline UI/html/newtrip.html]: @nziokaivy Blank line required at the end of the file

File: UI/html/profile.html [hunk 5]
```diff
@@ -171,7 +164,7 @@ <h1>Trips</h1>
                     <td><a href="specifictrip.html" class="more-button">More</a></td>
                 </tr>
             </tbody>
-        </table> 
-    </section> 
+        </table>
+    </section>
 </body>
-</html>
+</html>
```
Comment (oesukam) [inline UI/html/profile.html]: @nziokaivy Blank line required at the end of the file

File: UI/html/signup.html [hunk 6]
```diff
@@ -8,35 +8,34 @@
...
-        </div>
-    
+            <button class="btn">Sign Up</button>
+            <h5 class="signinlink">Already have an account ?
+                <a class="signinlink" href="signin.html">Sign In here</a>
+            </h5>
+        </form>
+    </div>
 </body>
 </html>
```
Comment (oesukam) [inline UI/html/signup.html]: @nziokaivy Blank line required at the end of the file

File: UI/html/specifictrip.html [hunk 7]
```diff
@@ -44,9 +44,8 @@
         <tr>
             <td>Bus License Number:</td>
             <td>KDE 463E</td>
-        </tr> 
+        </tr>
     </table>
-
     <button class="book-button">Book A Seat</button>
 </body>
 </html>
```
Comment (oesukam) [inline UI/html/specifictrip.html]: @nziokaivy Blank line required at the end of the file

File: UI/html/specifictrip_admin.html [hunk 8]
```diff
@@ -44,10 +44,8 @@
...
             <td>Bus License Number:</td>
             <td>KDE 463E</td>
-        </tr> 
+        </tr>
     </table>
-
     <button class="book-button">Cancel Trip</button>
-
 </body>
 </html>
```
Comment (oesukam) [inline UI/html/specifictrip_admin.html]: @nziokaivy Blank line required at the end of the file

**Review by oesukam — CHANGES_REQUESTED** [review]
@nziokaivy You are still using 4 spaces instead of 2 for tabulation

**Comment by oesukam** [comment]
@nziokaivy You forgot to add the text input after filter by dropdown. Don't forget to resolve conflict when you are done

**Comment by nziokaivy** [comment]
> @nziokaivy You forgot to add the text input after filter by dropdown. Don't forget to resolve conflict when you are done

@oesukam I intentionally left it out so that when I connect the front end to the backend it can receive information on which filter to display. i.e when I click on the origin in the dropdown, it will fetch all the trips by the origin from the backend.

**Pull request merged**